packages/apps/proof-of-us/src/components/Modal/style.css.ts

```diff
@@ -49,4 +49,5 @@ export const dialogClass = style([
 export const headerClass = style({
   fontSize: '18px',
   width: '100%',
+  fontWeight: '700',
```

Comment:
i think this is a value in atoms, no?

Reply:
But since atoms are wrappers, do we need to use them if we already have a custom class for the element?